cargo-rustc: add page (#4336)
tldr-pages/tldr: pages/common/cargo-rustc.md

# cargo rustc

> > Compile a package, and pass extra options to the compiler.

Reviewer: ```suggestion
> Compile a Rust package, and pass extra options to the compiler.
```

> More information: <https://doc.rust-lang.org/cargo/commands/cargo-rustc.html>.

> - Build the package or packages defined by the `Cargo.toml` manifest file in the local path:

Reviewer: ```suggestion
- Build the package or packages defined by the `Cargo.toml` manifest file in the current working directory:
```

`cargo rustc`

- Build artifacts in release mode, with optimizations:

`cargo rustc --release`

- Compile with architecture-specific optimizations for the current CPU:

`cargo rustc --release -- -C target-cpu=native`

- Compile with speed optimization:

`cargo rustc -- -C opt-level {{1|2|3}}`

> - Compile with size optimization:

Reviewer: ```suggestion
- Compile with [s]ize optimization (`z` also turns off loop vectorization):
```
